helpers/changesLogGenerator.js

```diff
@@ -0,0 +1,34 @@
+const { getTitle } = require('../constants');
+
+let issuesNumbersIncluded = [];
+
+function checkForDuplication(issue, duplicateIssues) {
+  if(duplicateIssues === true) return `\n- ${issue.title} [${issue.number}]`;
+
+  const duplicate = issuesNumbersIncluded.some(numberIncluded => (numberIncluded === issue.number));
+  if (!duplicate) {
+    issuesNumbersIncluded.push(issue.number);
+    return `\n- ${issue.title} [${issue.number}]`;
+  }
+}
+
+module.exports.generateChangesLog = function (groupIssues, {duplicateIssues, milestone}) {
```

Comment:
As previous, good to keep consistency and define the function outside of `module.exports`:

```suggestion
const generateChangesLog = (groupIssues, {duplicateIssues, milestone}) => {
```

Reply:
will add this as part of a single commit